test/test_write.c

```diff
@@ -28,21 +28,30 @@ This file is part of H5Z-ZFP. Please also read the BSD license
 
 #include "zfp.h"
 
-#ifdef ZFP_VERSION_MAJOR /* versioned release */
-#define ZFP_VERSION_LE(Maj,Min,Pat)  \
+#if defined(ZFP_VERSION_TWEAK)
+#define ZFP_VERSION_LE(Maj,Min,Pat,Twk)  \
+        (((ZFP_VERSION_MAJOR==Maj) && (ZFP_VERSION_MINOR==Min) && (ZFP_VERSION_PATCH==Pat) && (ZFP_VERSION_TWEAK<=Twk)) || \
+         ((ZFP_VERSION_MAJOR==Maj) && (ZFP_VERSION_MINOR==Min) && (ZFP_VERSION_PATCH<=Pat)) || \
+         ((ZFP_VERSION_MAJOR==Maj) && (ZFP_VERSION_MINOR<Min)) || \
+          (ZFP_VERSION_MAJOR<Maj))
+#elif defined(ZFP_VERSION_RELEASE) /* versioned release */
+#define ZFP_VERSION_LE(Maj,Min,Rel,Twk)  \
+        (((ZFP_VERSION_MAJOR==Maj) && (ZFP_VERSION_MINOR==Min) && (ZFP_VERSION_RELEASE<=Rel)) || \
+         ((ZFP_VERSION_MAJOR==Maj) && (ZFP_VERSION_MINOR<Min)) || \
+          (ZFP_VERSION_MAJOR<Maj))
+#elif defined(ZFP_VERSION_PATCH) /* versioned release */
+#define ZFP_VERSION_LE(Maj,Min,Pat,Twk)  \
         (((ZFP_VERSION_MAJOR==Maj) && (ZFP_VERSION_MINOR==Min) && (ZFP_VERSION_PATCH<=Pat)) || \
          ((ZFP_VERSION_MAJOR==Maj) && (ZFP_VERSION_MINOR<Min)) || \
-         (ZFP_VERSION_MAJOR<Maj))
-#else /* develop */
-#define ZFP_VERSION_LE(Maj,Min,Pat) 0
-#endif
-
-#if ZFP_HAS_CFP
-#if ZFP_VERSION_LE(0,5,5)
-#include "cfparrays.h"
+          (ZFP_VERSION_MAJOR<Maj))
 #else
-#include "cfparray.h"
+#error undefined ZFP library version symbols
 #endif
+
+#if !ZFP_VERSION_LE(0,9,9,9)
```

Comment:
I think you accidentally removed the line "#if ZFP_HAS_CFP"

Reply:
Oh, sheesh. yes. Thanks. Will fix momentarily